lib/galaxy/webapps/galaxy/api/cloud.py

```diff
@@ -233,6 +234,6 @@ def send(self, trans, payload, **kwargs):
                                                authz_id=authz_id,
                                                dataset_ids=dataset_ids,
                                                overwrite_existing=payload.get("overwrite_existing", False))
-        return {'sent_dataset_labels': sent,
-                'failed_dataset_labels': failed,
-                'bucket_name': bucket}
+        return json.dumps({'sent_dataset_labels': sent,
```

Comment:
The expose_api used here (and in the above method) should automatically be converting to json -- did you find it was not?

Reply:
That is what I guessed when implementing this in the first place, however, testing it with @Nerdinacan it turned out it the returned value is not exactly a JSON object.

Reply:
Interesting.  Can you debug what's going wrong with the expose_api decorator?

Reply:
The API returns an object like the following: 

```
{
    "sent_dataset_labels": [],
    "bucket_name": "vahid-objectstore-tests",
    "failed_dataset_labels": []
}
```

This looks a valid JSON to me, so, I guess maybe there was unrelated issue on the branch we were looking at.

Reply:
Ok, cool.  Let me know if you need help tracking it down, but I was going to be surprised/worried if the decorators were busted :)

Reply:
Sure, thanks. As things stand, we're good for now :)